code/campaignclassic/src/main/java/com/adobe/marketing/mobile/AEPPushNotificationBuilder.java

```diff
@@ -475,18 +475,20 @@ static void setVisibility(
      *
      * @param context the application {@link Context}
      * @param builder the notification builder
-     * @param actionButtons {@link List<AEPPushTemplate.ActionButton>} containing action buttons to
-     *     attach to the notification
+     * @param actionButtonsString {@code String} a JSON string containing action buttons to attach
```

Comment:
Does this need to be a string (instead of a JSON object) to allow for incorporating this data into Intents in addition to the template?

Reply:
that is correct